test/helper.rb

```diff
@@ -44,10 +44,6 @@ class TestCase < MiniTest::Test
     extend LiveServerConfig
     self.live_server = ENV['LIVE']
 
-    def test_default
```

Comment:
I couldn't find a single reference to this, and the trailing modifier can not be used to prevent method definition, so it was actually not doing anything.